ui/app/manager/src/pages/page-export.ts

```diff
@@ -200,6 +204,14 @@ export class PageExport extends Page<AppStateKeyed> {
     @property({type: Number})
     private latestTimestamp: number = moment().valueOf();
 
+    @property()
+    protected _loading: boolean = false;
+
+    protected _exportFormats: {[key: number]: string} = {1: 'CSVDefault', 2: "CSVCrosstab", 3: "CSVCrosstab | 1min interval"};
```

Comment:
I'd try to use an enum for this.

As Rich proposed that for the backend as well, I would simply re-use that one here,
and update both the `_exportFormats` variable and `selectedFormat` property.